template/docker-compose.yml.tmpl

```diff
@@ -25,8 +25,10 @@ services:
       "-cadvisor", "http://cadvisor:8080",
       {{ if .NeedsRelay -}}
       "-fakerelay", "http://relay:5000",
-      {{- end }}
+      "-containers", "app-{{ .RunName }}-{{ .ID }},fakerelay-{{ .RunName }}-{{ .ID }},postgres-{{ .RunName }}-{{ .ID }}",
```

Comment:
The database should be outside of the conditional since it is deployed in both cases.

I'd add a new `if relay` inside of the list of containers.